test/e2e/anti_affinity_test.go

```diff
@@ -105,6 +105,9 @@ func VerifyAntiAffinity(ctx context.Context, input AntiAffinitySpecInput) {
 		WaitForClusterIntervals:      input.Global.E2EConfig.GetIntervals("", "wait-cluster"),
 		WaitForControlPlaneIntervals: input.Global.E2EConfig.GetIntervals("", "wait-control-plane"),
 		WaitForMachineDeployments:    input.Global.E2EConfig.GetIntervals("", "wait-worker-nodes"),
+		PostMachinesProvisioned: func() {
```

Comment:
Should this be PostClusterInitialized?
(with some tolerance to give time to CRS to be applied, assuming we are using CRS to deploy the CSI/CPI)

Reply:
I was not able to find a `PostClusterInitialized` 🤔

Reply:
Note: have to rethink this, `PostMachinesProvisioned` won't be very helpful because this is more on a happy path then. Machines may not even get provisioned and we want the logs.